velox/docs/functions/spark/json.rst

```diff
@@ -2,9 +2,43 @@
 JSON Functions
 ==============
 
+JSON Format
+-----------
+
+JSON is a language-independent data format that represents data as
+human-readable text. A JSON text can represent a number, a boolean, a
+string, an array, an object, or a null, with slightly different grammar.
```

Comment:
> with slightly different grammar

nit: perhaps add the target to compare against.

Reply:
There are some examples listed following this statement.

Reply:
@rui-mo, just remove this unnecessary statement: `with slightly different grammar`. The description for certain types' representation is enough.